### ISSUE TITLE
Removal of entire development team

### ISSUE THREAD

**Issue by PartTimeLegend** [buttercoin/buttercoin#71]
As a previous member of this development project who was removed without any prior notice. I would love to see an explanation as to why we were all removed.

That coupled with the 23.01259287 BTC currently being held for this project. Which amounts to $ 13,692.49 which I can only see as being stolen by the remaining single developer.

Whilst the money is not an issue, it was designed to pay back the developers for their time and effort. However no remuneration was ever made to developers kicked from the project without any notice.

An explanation would be nice.

**Comment by Marak** [comment]
+1

I am also in the same situation as @PartTimeLegend.

It's really shameful what the creators of this project did in terms of being dishonest with the open-source community. I spent 100s of hours getting Buttercoin off the ground last year. It was my hard work which allowed the project to raise 1,000,000 dollars in cash.

Thanks guys.

**Comment by Marak** [comment]
For anyone unfamiliar you can always check the contribution graph on the project.

https://github.com/buttercoin/buttercoin/graphs/contributors

<img src="http://i.imgur.com/mdIs2sO.png"/>

**Comment by RevCBH** [comment]
I don't know why anyone was removed from the public projects, it certainly wasn't intentional and I'd be happy to add you back. We aren't actively using or maintaining any of the node code, so we have no reason to try and take access away from anyone that was still interested in that codebase.

Regarding the donations, I've offered to return donated funds to anyone that can demonstrate they control an address that sent in a transaction. Nobody has tried to claim any, so it's all just been sitting there. If you've donated and want me to send you your coins back, let me know and we'll get it sorted out. The donations were collected without a clear purpose, so it doesn't seem right to do anything with them besides return them.

**Comment by PartTimeLegend** [comment]
You don't know why anyone was removed? As repository owner I find it hard to believe as you would have been the person doing so.

If the project is abandoned and the funds raised were to fund the developers' time. Then I would personally look at distributing the funds between the developers on the project. As that is what they were originally raised for.

**Comment by RevCBH** [comment]
Correct, meaning if you were removed it was by mistake. I am not the only person with "owner" privileges on the organization. I, and others, were recently reorganizing permissions on some projects and could have dropped people. There was no malice in it. I've restored permissions on the original repos, please let me know if I missed any.

I didn't solicit them, but It was my understanding that the donated funds would be used for many things, like maintaining infrastructure, paying for feature or bug bounties, etc. If you can point me to an explicit decision on how to disburse the funds, I will act accordingly. In the meantime, I only feel comfortable returning them to the original donators.

**Comment by PartTimeLegend** [comment]
Looks like yet again I have been removed from the development team.

I call out another scan artist this time @kevin-buttercoin, Contrary to the scammer @RevCBH who claimed it was by mistake @kevin-buttercoin did this intentionally to try to hide his scam.

I know that @RevCBH is a holy man at least in title and I hope he enjoys his time on the rack in the fire pits of hell.

@kevin-buttercoin I believe you are the man with the private key and the ability to end this whole debate. You won't because you are dishonest man.
